src/main/io/osd.c

```diff
@@ -2170,6 +2173,12 @@ static bool osdDrawSingleElement(uint8_t item)
             break;
         }
 
+    case OSD_GFORCE:
+        {
+            osdFormatCentiNumber(buff, (float)(imuMeasuredAccelBF.v[2]/GRAVITY_MSS), 0, 1, 0, 3);
```

Comment:
I believe a modulus of `imuMeasuredAccelBF` should be more meaningful. I.e. for a plane in a banked turn some of acceleration is applied on a lateral axis.

Reply:
There are several way to do that feature, of course. I think, the simplest is the one I introduced (to learn some iNav code), but of course there may be better ideas. I think better option would be to get max of abs of all x,y,z but with display the result as not abs but with actual sign. This make sense since accel readings are normalized. I think this idea contains what you wrote, digitalentity.